dashboard/test/controllers/ops/workshop_attendance_controller_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 module Ops
-  class WorkshopAttendanceControllerTest < ::ActionController::TestCase
-    include Devise::Test::ControllerHelpers
+  class WorkshopAttendanceControllerTest < ::ActionDispatch::IntegrationTest
+    include Devise::Test::IntegrationHelpers
```

Comment:
Some tests (such as this one) needed to be converted to `ActionDispatch::IntegrationTest` in order to continue working as intended, refactoring the request-actions from e.g.:
```
method :action, param: 'value'
```
to:
```
method 'path', params: {param: 'value'}
```
(I don't know exactly why- something changed slightly in the behavior of `ActionController::TestCase`, which is now deprecated anyway.)